src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentRenderModeDirectiveIntegrationTests.cs

```diff
@@ -128,9 +128,9 @@ @rendermode Microsoft.AspNetCore.Components.Web.RenderMode.Server
 
         var assemblyResult = CompileToAssembly(compilationResult, throwOnFailure: false);
         assemblyResult.Diagnostics.Verify(
-            // x:\dir\subdir\Test\TestComponent.cshtml(1,3): error CS0103: The name 'rendermode' does not exist in the current context
+            // x:\dir\subdir\Test\TestComponent.cshtml(1,2): error CS0103: The name 'rendermode' does not exist in the current context
             // __builder.AddContent(0, rendermode);
-            Diagnostic(ErrorCode.ERR_NameNotInContext, "rendermode").WithArguments("rendermode").WithLocation(1, 3)
+            Diagnostic(ErrorCode.ERR_NameNotInContext, "rendermode").WithArguments("rendermode").WithLocation(1, 2)
```

Comment:
@dotnet/razor-compiler looks like the Roslyn change also broke this test. Is that expected?

Seems to me like the new position is correct though, and previously the squiggle would have started at the "e" in "rendermode", unless the position is 0-based. But if it was 0-based, the above test would have a squiggle starting at the "o" in "NoExist"

Reply:
Yes, that's expected, it's due to this bug fix: https://github.com/dotnet/roslyn/pull/70247

Reply:
We should now be also able to re-enable disabled tests thanks to that - you can cherry-pick https://github.com/dotnet/razor/pull/9447/commits/9d1bd18bd3a736297e1a596fa1233b01dee93e69 or I will open a follow up PR when this PR is merged.

Reply:
re-enabled.